ls1intum/Artemis
# Bugfix: Fix TypeError: Cannot read property 'name' of undefined

### Checklist

### Description
The current tour implementation does not do error handling when trying to look for the highlighted element through `document.querySelector()`. This is why we get the `ERROR TypeError: Cannot read property 'name' of undefined` when an HTML element cannot be found with a certain selector, see:
![image](https://user-images.githubusercontent.com/33764166/64527195-96cfaa80-d305-11e9-9535-209629bc423b.png)

### Steps for Testing
1. Log in to Artemis
2. Navigate to Course Exercise Overview Page, e.g. https://artemistest.ase.in.tum.de/#/overview/19/exercises
3. Make sure that all exercises are hidden (by clicking the exercise rows)
4. Open browser console
5. Start tour by selecting the 'Start tour' item from the account menu
6. The TypeError should not be visible in the console log

### Screenshots
![image](https://user-images.githubusercontent.com/33764166/64527746-01351a80-d307-11e9-8405-a04354ecefad.png)
